### PR TITLE
Create multion_news_agent.py

### PR DESCRIPTION
**Pull request by Div99**
### Type of Change

- [X ] New Example
- [ ] Example updates (Bug fixes, new features, etc.)
- [ ] Other (changes to the codebase, but not to examples)

## Checklist

- [X ] Example is testable in synthetic monitoring system, or `lambda-test: false` is added to example frontmatter
- [ ] Example does _not_ require third-party dependencies to be installed locally
- [ ] Example is documented with comments throughout, in a [_Literate Programming_](https://en.wikipedia.org/wiki/Literate_programming) style.

## Outside contributors

You're great! Thanks for your contribution.

### CODE REVIEW

**Comment by Div99** [comment]
@aksh-at

File: 10_integrations/multion_news_agent.py [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+from datetime import datetime, timedelta
+
+import modal
+
+stub = modal.Stub("ai-news-tweet-agent")
+
+# ## Searching for AI News
+#
+
+QUERY = "serverless"
```
Comment (aksh-at) [inline 10_integrations/multion_news_agent.py:22]: I don't think these constants are used anywhere

File: 10_integrations/multion_news_agent.py [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+WINDOW_SIZE_DAYS = 1
+
+# Let's also define an image that has the `requests` package installed, so we can query the API.
```
Comment (aksh-at) [inline 10_integrations/multion_news_agent.py:25]:
```suggestion
# Let's also define an image that has the `muliton` package installed, so we can query the API.
```

File: 10_integrations/multion_news_agent.py [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+
+multion_image = modal.Image.debian_slim().pip_install("multion")
+
+# We can now define our main entrypoint, that use MultiOn to scrape AI news everyday and post it on my twitter account. We specify a [schedule](/docs/guide/cron) in the function decorator, which
```
Comment (aksh-at) [inline 10_integrations/multion_news_agent.py:29]:
```suggestion
# We can now define our main entrypoint, that uses [MultiOn](https://www.multion.ai/) to scrape AI news everyday and post it on our twitter account. We specify a [schedule](/docs/guide/cron) in the function decorator, which
```

Comment (aksh-at) [inline 10_integrations/multion_news_agent.py:29]: Can we add a section that explains what MultiOn is, and how to set it up for this example? I.e. I have to make an account, install a chrome extension, log in to my Twitter (?)

**Review by aksh-at — CHANGES_REQUESTED** [review]
great, thanks for adding this! Couple comments but main one is that it would be great to explain to users what MultiOn is and how to set it up :)

**Review by aksh-at — APPROVED** [review]
Looks like there's a couple typos, but good to go otherwise!

**Pull request merged**